.travis.yml

```diff
@@ -14,7 +14,13 @@ jdk:
   - oraclejdk8
 
 script:
-  - ./gradlew clean build -x checkstyleTest --stacktrace
+  - ./gradlew build connectedCheck -x checkstyleTest --stacktrace
```

Comment:
That's what I thought too but according to http://tools.android.com/tech-docs/new-build-system/user-guide

> build does not depend on deviceCheck, or connectedCheck.